src/components/step/YouTubeStep/YouTubeStep.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react'
+import MyDataHelps, {StepConfiguration} from '@careevolution/mydatahelps-js';
+import StepTitle from '../StepTitle';
+import StepText from '../StepText';
+import StepDetailText from '../StepDetailText'
+import StepNextButton from '../StepNextButton';
+import '../step.css';
+
+export interface YouTubeStepProps {
+    properties?: any;
+    styles?: any;
+}
+
+export default function (props: YouTubeStepProps) {
+
+    const [properties, setProperties] = useState<any>(props.properties ?? {});
+    const [styles, setStyles] = useState<any>(props.styles ?? {});
+
+    useEffect(() => {
+        MyDataHelps.getStepConfiguration().then(function(config: StepConfiguration){
+          if (!config) return; // allows test mode to work
+          setProperties(config?.properties ?? {});
+          setStyles(config?.styles ?? {});
+        });
+    }, []);
+
+    if (!properties.videoId) {
+        return <div className="step-container"></div>;
+    }
+    // ref player params at https://developers.google.com/youtube/player_parameters
+    let frameHeight = (properties.height ? properties.height : "225") + "px";
+    let ccLanguage = MyDataHelps.getCurrentLanguage(); // used to set default CC language
+    let showRelated = 0; // restricts related videos to current channel only
+    let showControls = 1; // show player controls, probably something we should make configurable
+    let autoplay = 0; // do not autoplay, probably should make this configurable
+    let allowFullscreen = 1; // show fullscreen controls
+    let youTubeBranding = 1; // hide youtube logo
+    return (
+      <div className="step-container">
+           <StepTitle 
+              text={properties.title} 
+              textAlign={styles.titleAlignment}
+              color={styles.titleColor}
+              fontSize={styles.titleFontSize}
+              fontWeight={styles.titleFontWeight}
+            />
+           <StepText 
```

Comment:
Something weird here I can't figure out:
![image](https://user-images.githubusercontent.com/1643171/215774104-c9854ce9-4b74-4c73-8978-fdf4d482018e.png)

The step text appears to have some extra height on it.  Looking at chrome browser tools I can't really figure out where it's coming from though (the inner <p> element has the right height)

Reply:
This was caused by interaction between markdown and the "pre" whitespace styling. Took a while to get rid of it but basically just needed to trim what is coming back from the markdown library.